packages/utils/src/saml/bulk-saml.ts

```diff
@@ -0,0 +1,81 @@
+// This is a helper script that lets you test constructing your own soap+saml requests. It creates the SOAP Envelope and then sends it to the gateway specified in the request body.
+// npm run saml-server and then reference the Metriport- IHE GW / XML + SAML Constructor - Postman collection
+import * as dotenv from "dotenv";
+dotenv.config();
+import fs from "fs";
+import { v4 as uuidv4 } from "uuid";
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import { XCPDGateway } from "@metriport/ihe-gateway-sdk";
+import { createAndSignBulkXCPDRequests } from "@metriport/core/external/carequality/ihe-gateway-v2/outbound/xcpd/create/iti55-envelope";
+import { sendSignedXCPDRequests } from "@metriport/core/external/carequality/ihe-gateway-v2/outbound/xcpd/send/xcpd-requests";
+import { processXCPDResponse } from "@metriport/core/external/carequality/ihe-gateway-v2/outbound/xcpd/process/xcpd-response";
+
+const samlCertsAndKeys = {
+  publicCert: getEnvVarOrFail("CQ_ORG_CERTIFICATE_PRODUCTION"),
+  privateKey: getEnvVarOrFail("CQ_ORG_PRIVATE_KEY_PRODUCTION"),
+  privateKeyPassword: getEnvVarOrFail("CQ_ORG_PRIVATE_KEY_PASSWORD_PRODUCTION"),
+  certChain: getEnvVarOrFail("CQ_ORG_CERTIFICATE_INTERMEDIATE_PRODUCTION"),
+};
+
+const patientId = uuidv4();
+const cxId = uuidv4();
+
+const bulkPath = "/Users/jonahkaye/Desktop/MetriportUnicorn/metriport/scratch/girth.json";
+async function main() {
+  const body = JSON.parse(fs.readFileSync(bulkPath, "utf8"));
+
+  body.gateways = body.gateways.filter(
+    (gateway: XCPDGateway) =>
+      !gateway.url.includes("https://rle.surescripts.net/IHE/PatientDiscovery") &&
```

Comment:
surescripts only has 1 gateway:
https://rle.surescripts.net/IHE/gateway/xcpd
The remaining 36 endpoints in the directory structured like the above do not work. [Context](https://metriport.slack.com/archives/C065BLRBUDQ/p1714949449094709):